remove assertion when disabling floating-point extensions does not work

--- a/machine/minit.c	2019-05-21 12:27:24.267206118 +0200
+++ b/machine/minit.c	2019-05-21 12:26:42.226980668 +0200
@@ -77,7 +77,7 @@
 #else
   uintptr_t fd_mask = (1 << ('F' - 'A')) | (1 << ('D' - 'A'));
   clear_csr(misa, fd_mask);
-  assert(!(read_csr(misa) & fd_mask));
+  //assert(!(read_csr(misa) & fd_mask));
 #endif
 }
 
